src/main/java/bio/terra/app/controller/ConfigsApiController.java

```diff
@@ -0,0 +1,148 @@
+package bio.terra.app.controller;
+
+import bio.terra.app.configuration.ApplicationConfiguration;
+import bio.terra.controller.ConfigsApi;
+import bio.terra.model.ConfigEnableModel;
+import bio.terra.model.ConfigGroupModel;
+import bio.terra.model.ConfigListModel;
+import bio.terra.model.ConfigModel;
+import bio.terra.service.configuration.ConfigurationService;
+import bio.terra.service.dataset.AssetModelValidator;
+import bio.terra.service.iam.AuthenticatedUserRequest;
+import bio.terra.service.iam.AuthenticatedUserRequestFactory;
+import bio.terra.service.iam.IamAction;
+import bio.terra.service.iam.IamResourceType;
+import bio.terra.service.iam.IamService;
+import bio.terra.service.iam.PolicyMemberValidator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.Api;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+import java.util.Optional;
+
+@Controller
+@Api(tags = {"configs"})
```

Comment:
Is this tag necessary? I though it was sufficient to implement the generated API. At least that's what I've always done for spring apps, and it seems to work fine.

Reply:
Its necessary to de-duplicate the generated groups in the Swagger UI. The tag links the <ENDPOINT>ApiController to the <ENDPOINT> tag in the OpenAPI yaml.